### PR TITLE
Create Region Channel Plan chain vars

### PR DESCRIPTION
**Pull request by mikev**

### CODE REVIEW

**Comment by mikev** [comment]
Encode the LoRaWan Region channel plans into the blockchain.  This is a more full-featured effort than the existing region_param.
It will allow us to create a Region library and simplify the existing Router, Miner and light gateway code.

File: src/blockchain_channel_plan_v1.proto [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  message dr_range {
+    uint32 min_dr = 1;
+    uint32 max_dr = 2;
+  }
+
+  channel_plan_id id = 1;
+  string name = 2;
+  region region = 3;
+  bool is_dynamic = 4;
+  float min = 5;
```
Comment (vihu) [inline src/blockchain_channel_plan_v1.proto:88]: nitpicking, these can be a bit more descriptive perhaps?

**Comment by madninja** [comment]
@mikev there was no further explanation or activity around this PR. Can this get closed?

**Comment by madninja** [comment]
No activity, closing.. @mikev re-open if this needs re-activating

**Pull request closed without merging**